packages/@lwc/signals/src/signal-identity.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+let signalIdentity: symbol | undefined;
+
+export function setSignalIdentity(identity: symbol) {
+    signalIdentity = identity;
+}
+
+export function getSignalIdentity(): symbol | undefined {
+    return signalIdentity;
+}
```

Comment:
You can just import this from `@lwc/shared` and then add `@lwc/shared` to the `devDependencies` of `@lwc/signal`'s `package.json`. Then the function will be bundled into the `dist/index.js`.

Reply:
Moved to using a trusted signal registry approach